warehouse/static/sass/blocks/_button-set.scss

```diff
@@ -0,0 +1,26 @@
+.button-set {
+  display: inline-block;
+
+  > .button {
+    display: inline;
+    float: left;
+    margin-left: -2px;
+    border-radius: 0;
+
+    &:first-child {
+      border-radius: 3px 0 0 3px;
+    }
+
+    &:last-child {
+      border-radius: 0 3px 3px 0;
+      z-index: 5;
```

Comment:
hacky, had to create to resolve the overlapping border (grey color) of the last element while on the last page.